lib/bucky/test_equipment/pageobject/base_pageobject.rb

```diff
@@ -8,6 +8,21 @@ module TestEquipment
     module PageObject
       class BasePageObject
         include Bucky::Utils::YamlLoad
+
+        # https://www.rubydoc.info/gems/selenium-webdriver/2.53.0/Selenium/WebDriver/SearchContext#find_elements-instance_method
```

Comment:
# https://www.rubydoc.info/gems/selenium-webdriver/3.142.0/Selenium/WebDriver/SearchContext#find_elements-instance_method
Please change to '3.142.0'

Reply:
Thanks for your advice.
I found official documents from https://github.com/SeleniumHQ/selenium/tree/master/rb.

https://seleniumhq.github.io/selenium/docs/api/rb/Selenium/WebDriver/SearchContext.html#find_element-instance_method